controllers/chat/ChatController.js

```diff
@@ -446,7 +570,7 @@ module.exports = {
         error: validated
       });
     const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
-    const {members} = req.body;
+    const {members, oldChannelId} = req.body;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The destructuring assignment in the request body has been updated to include `oldChannelId`. This is used when creating a new chat channel to retrieve anonymous user info from an old channel if it exists.
```diff
- const {members} = req.body;
+ const {members, oldChannelId} = req.body;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->